lib/dotenv/rails.rb

```diff
@@ -4,7 +4,6 @@
   return
 end
 
-require "dotenv"
```

Comment:
This  require statement shouldn't be necessary since it is already required as the entrypoint of the gem. I suspect that it is still there from the time that Rails integration was a separate gem. Removing this gets rid of the warning in my app and does not seem to break the specs in the dotenv gem.